### PR TITLE
Update performance scaling test rig

### PR DESCRIPTION
**Pull request by rzats**
**Prerequisites**:

- [X] Unless it is a documentation hotfix it should be merged against the `dev` branch
- [X] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [X] Build is successful
- [X] Code is cleaned up and formatted

### Summary

Adds several updates to the performance scaling test rig:
- Now clones `cmu-delphi/delphi-admin` repo, so files do not need to be duplicated to this repo.
  - The test files added to this repo have been deleted accordingly.
- Changes the approach to selecting queries performance testing - instead of running queries from a small sample for a fixed amount of time, runs every query from a larger sample once; this is more consistent across multiple runs.
- Writes Locust output to an artifact in the GitHub Actions run.

### CODE REVIEW

File: .github/workflows/performance-tests.yml [hunk 1]
```diff
@@ -52,11 +52,29 @@ jobs:
...
           cd ../driver
           sudo make web sql="${{ secrets.DB_CONN_STRING }}"
+      - name: Check out delphi-admin
+        uses: actions/checkout@v3
+        with:
+          repository: cmu-delphi/delphi-admin
+          token: ${{ secrets.GH_PERSONAL_TOKEN }}
+          path: delphi-admin
       - name: Build & run Locust
+        continue-on-error: true # sometimes ~2-5 queries fail, we shouldn't end the run if that's the case
```
Comment (melange396) [inline .github/workflows/performance-tests.yml:62]: what are the failures?  they only happen transiently?

**Review by melange396 — APPROVED** [review]
looks pretty good!

on the wishlist: it might be nice to have summary info about the results of the run included in the follow-up comment, like the running time of the `docker run locust` command or a count/percent of the failed queries.  [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idoutputs) looks like a good way to get information passed between the jobs.

File: .github/workflows/performance-tests.yml (hunk 1 above)
Comment (rzats) [reply .github/workflows/performance-tests.yml:62]: There are two queries from the "v4 - small" set that consistently fail.

File: .github/workflows/performance-tests.yml [hunk 2]
```diff
@@ -52,11 +52,29 @@ jobs:
         run: |
           cd ../driver
           sudo make web sql="${{ secrets.DB_CONN_STRING }}"
+      - name: Check out delphi-admin
+        uses: actions/checkout@v3
+        with:
+          repository: cmu-delphi/delphi-admin
+          token: ${{ secrets.GH_PERSONAL_TOKEN }}
```
Comment (korlaxxalrok) [inline .github/workflows/performance-tests.yml:59]:
```suggestion
          token: ${{ secrets.CMU_DELPHI_DEPLOY_MACHINE_PAT }}
```

**Review by korlaxxalrok — APPROVED** [review]
LGTM 👍

File: .github/workflows/performance-tests.yml [hunk 3]
```diff
@@ -52,11 +52,29 @@ jobs:
...
+          token: ${{ secrets.CMU_DELPHI_DEPLOY_MACHINE_PAT }}
+          path: delphi-admin
       - name: Build & run Locust
+        continue-on-error: true # sometimes ~2-5 queries fail, we shouldn't end the run if that's the case
         run: |
-          cd ../driver/repos/delphi/delphi-epidata/tests/performance
+          cd delphi-admin/load-testing/locust
           docker build -t locust .
-          docker run --net=host -v $PWD:/mnt/locust -e CSV=/mnt/locust/v4-requests-smaller.csv locust -f /mnt/locust/v4.py --host http://127.0.0.1:10080/ --users 10 --spawn-rate 1 --headless -t 15m
+          export CSV=v4-requests-small.csv
```
Comment (krivard) [inline .github/workflows/performance-tests.yml:66]: question: why move the CSV filename into a variable?

(is it a stub for being able to override it in the future?)

**Review by krivard — APPROVED** [review]
👍 happy to have this merged as-is; question is largely out of curiosity

File: .github/workflows/performance-tests.yml (hunk 3 above)
Comment (rzats) [reply .github/workflows/performance-tests.yml:66]: It's just to avoid repeating the same filename twice, since it's now used in the `wc -l` bit as well.

**Pull request merged**